build.sbt

```diff
@@ -2,27 +2,27 @@
 resolvers += "Typesafe Releases" at "http://repo.typesafe.com/typesafe/releases/"
 
 val scalaCheck = "org.scalacheck" %% "scalacheck" % "1.14.0"
-val specs2 = "org.specs2" %% "specs2-core" % "4.3.2"
+val specs2 = "org.specs2" %% "specs2-core" % "4.4.1"
 val scalaTest = "org.scalatest" %% "scalatest" % "3.0.5"
-val mockito = "org.mockito" % "mockito-core" % "2.21.0"
-val akkaStreamTestKit = "com.typesafe.akka" %% "akka-stream-testkit" % "2.5.14"
+val mockito = "org.mockito" % "mockito-core" % "2.24.5"
+val akkaStreamTestKit = "com.typesafe.akka" %% "akka-stream-testkit" % "2.5.21"
 
-val snakeYaml =  "org.yaml" % "snakeyaml" % "1.21"
+val snakeYaml =  "org.yaml" % "snakeyaml" % "1.23"
 val commonsIO = "commons-io" % "commons-io" % "2.6"
-val commonsCodec = "commons-codec" % "commons-codec" % "1.11"
-val bouncyCastle = "org.bouncycastle" % "bcpkix-jdk15on" % "1.60"
+val commonsCodec = "commons-codec" % "commons-codec" % "1.12"
+val bouncyCastle = "org.bouncycastle" % "bcpkix-jdk15on" % "1.61"
 
 // the client API request/response handing uses Akka Http 
-val akkaHttp = "com.typesafe.akka" %% "akka-http" % "10.1.3"
-val akkaStream = "com.typesafe.akka" %% "akka-stream" % "2.5.14"
-val akka = "com.typesafe.akka" %% "akka-actor" % "2.5.14"
+val akkaHttp = "com.typesafe.akka" %% "akka-http" % "10.1.7"
+val akkaStream = "com.typesafe.akka" %% "akka-stream" % "2.5.21"
+val akka = "com.typesafe.akka" %% "akka-actor" % "2.5.21"
 
 // Skuber uses akka logging, so the examples config uses the akka slf4j logger with logback backend
-val akkaSlf4j = "com.typesafe.akka" %% "akka-slf4j" % "2.5.14"
-val logback = "ch.qos.logback" % "logback-classic" % "1.1.3" % Runtime
+val akkaSlf4j = "com.typesafe.akka" %% "akka-slf4j" % "2.5.21"
+val logback = "ch.qos.logback" % "logback-classic" % "1.2.3" % Runtime
 
 // the Json formatters are based on Play Json
-val playJson = "com.typesafe.play" %% "play-json" % "2.6.9"
+val playJson = "com.typesafe.play" %% "play-json" % "2.7.1"
```

Comment:
I assume this change could cause binary incompatibility issues for users of skuber who are also using Play 2.6.x in their application, unless they change that dependency to Play 2.7.x? (If so, I would target this for a minor (v2.2) rather than patch (v2.2.2) release, a general principal with skuber is not to have binary incompatibilities across patch releases.)

Reply:
Yes. This is an ongoing problem when pulling in Play ... it is a framework and not a library.  Could try shading it or using a different http layer that is less opinionated.